.github/workflows/labels.yml

```diff
@@ -1,49 +1,83 @@
-name: Required Labels
+name: Required Labels and Breaking Change Detection
 
 on:
   pull_request:
-    types: [opened, labeled, unlabeled, synchronize]
+    types: [opened, labeled, unlabeled, synchronize, edited]
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
 jobs:
-  label:
```

Comment:
fwiw this is now blocking the PR as this was the name of the required CI. 
So you'll need to update the branch protection rules to merge this PR.